automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/ProfileSpec.scala

```diff
@@ -27,5 +28,18 @@ class ProfileSpec extends FreeSpec with WebBrowserSpec with UserFixtures with Ma
 
       }
     }
+
+    "should link with fence" in withWebDriver { implicit driver =>
+      val user = UserPool.chooseStudent
+      withSignIn(user) { _ =>
+        val profilePage = new ProfilePage().open
+
+        profilePage.linkFence
```

Comment:
I think this should differentiate between the different providers.  So I think that means we would need to generate the `data-test-id` to have the provider name included, and then your tests should do the same.